Pastebin.js

```diff
@@ -0,0 +1,179 @@
+{
+	"translatorID": "1d1a51d4-60bf-44b8-ae28-8b154d1ed721",
+	"label": "Pastebin",
+	"creator": "febrezo",
+	"target": "https?://(www.)?pastebin.com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 1,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-09-02 19:15:59"
+}
+
+/*
+	Pastebin Translator
+	Copyright (C) 2016 Félix Brezo, felixbrezo@gmail.com
+	
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the Affero GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+	
+	You should have received a copy of the Affero GNU General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+function detectWeb(doc, url) {
+	Zotero.debug("Starting detectWeb...");
+	return "webpage";
+}
+
+// Detecting the type of URL
+function getType(url) {
+	if ( url.indexOf('/u/')>-1 ) {
+		Zotero.debug("This is a profile...");
+		return "profile";
+	} else  	{
+		Zotero.debug("This is a paste");
+		return "paste";
+	} 
+}
+
```

Comment:
In Pastebin you can generally find two type of `webpages`: pastes themselves and, for the registered profiles, profile pages with some extra information about the profile. Metadata is extracted slightly differently in both pages so I coded this simple function to find out what type of page the user is viewing. Anyway, I can include that info inside doWeb.